CordovaLib/Classes/Private/Plugins/CDVLocalStorage/CDVLocalStorage.m

```diff
@@ -23,19 +23,12 @@ Licensed to the Apache Software Foundation (ASF) under one
 @interface CDVLocalStorage ()
 
 @property (nonatomic, readwrite, strong) NSMutableArray* backupInfo;  // array of CDVBackupInfo objects
-#if !WK_WEB_VIEW_ONLY
-@property (nonatomic, readwrite, weak) id <UIWebViewDelegate> webviewDelegate;
-#endif
 
 @end
 
 @implementation CDVLocalStorage
```

Comment:
I'm pretty sure that all the paths in this file for LocalStorage are UIWebView-specific. Worth a bit more investigation to be sure, but maybe this can be removed entirely.